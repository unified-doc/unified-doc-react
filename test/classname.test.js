import React from 'react';
import renderer from 'react-test-renderer';

import Doc from '..';
import { htmlContent } from './fixtures';

describe('classname', () => {
  it('applies custom CSS class name', () => {
    const tree = renderer
      .create(
        <Doc
          className="custom-doc"
          content={htmlContent}
          filename="doc.html"
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
