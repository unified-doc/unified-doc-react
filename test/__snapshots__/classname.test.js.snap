// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`classname applies custom CSS class name 1`] = `
<div
  className="custom-doc"
>
  <div>
    <blockquote>
      <strong>
        some
      </strong>
      
content
    </blockquote>
  </div>
</div>
`;
